test/js/web/fetch/fetch.brotli.test.ts

```diff
@@ -0,0 +1,25 @@
+import { test, expect } from "bun:test";
+
+test("fetch brotli response works", async () => {
+  const [firstText, secondText, { headers }] = await Promise.all([
+    fetch("https://bun.sh/logo.svg", {
```

Comment:
Can we use `example.com` instead?

Reply:
no, example.com doesn't return with brotli